README.rst

```diff
@@ -14,6 +14,58 @@ Contributing
 Please read `CONTRIBUTING.md <https://github.com/aws/sagemaker-mxnet-training-toolkit/blob/master/CONTRIBUTING.md>`__
 for details on our code of conduct, and the process for submitting pull requests to us.
 
+Testing
+-------
+
+Set up a virtual environment for testing.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+One of the multiple ways to setup a virtual environment
+
+::
+
+   # use a package virtualenv
+   # create a virtualenv
+   virtualenv -p python3 <name of env>
+   # activate the virtualenv
+   source <name of env>/bin/activate
+
+Install requirements
+~~~~~~~~~~~~~~~~~~~~
+
+::
+
+   pip install .[test]
+
+Install sagemaker-mxnet-training-toolkit
```

Comment:
previous step already installs 'sagemaker-mxnet-training-toolkit'

Reply:
And we actually configured tox for unit test: `tox -e py27,py36,py37 --parallel all -- test/unit`, which will take care of dependencies in different py versions

Reply:
Thanks for pointing it out. Addressed